tests/adapter/mongo/test_case_handling.py

```diff
@@ -601,3 +601,19 @@ def test_get_cases_cohort(real_adapter, case_obj, user_obj):
     result = adapter.cases(name_query="cohort:{}".format(cohort_name))
     # THEN we should get the case returned
     assert sum(1 for i in result) == 1
+
+def test_get_cases_solved_since(real_adapter, case_obj, user_obj, institute_obj, variant_obj):
```

Comment:
I know it would be a bit of a pain, but could you include a small test for each of the options usable in the query that can be combined with the time frame?